CHANGELOG.md

```diff
@@ -4,6 +4,58 @@ All notable changes to this project will be documented in this file.
 
+## [1.1.0] - 2022-08-19
+
+### Features
+
+- [`worker`] Add webpack configuration, closes [#537](https://github.com/temporalio/sdk-typescript/issues/537) ([#815](https://github.com/temporalio/sdk-typescript/pull/815))
+
+  This was our most-upvoted feature request! ([9 👍's](https://github.com/temporalio/sdk-typescript/issues?q=is%3Aissue+sort%3Areactions-%2B1-desc).) See [`WorkerOptions.bundlerOptions.webpackConfigHook`](https://typescript.temporal.io/api/interfaces/worker.workeroptions/#bundleroptions) for usage.
+
+- Add `@temporalio/nyc-test-coverage` package ([#798](https://github.com/temporalio/sdk-typescript/pull/798), thanks to [`@vkarpov15`](https://github.com/vkarpov15) 🙏)
+
+  This package adds code coverage for Istanbul. It's currently in beta: the API may be unstable as we gather feedback on it from users. To try it out, see [this code snippet](https://github.com/temporalio/sdk-typescript/pull/798#issue-1323652976) for current usage.
+
+- [`common`] Improve `ApplicationFailure` arguments; add `.create` and `.fromError` ([#767](https://github.com/temporalio/sdk-typescript/pull/767))
+
+  See [`ApplicationFailure.create`](https://typescript.temporal.io/api/classes/common.applicationfailure/#create) and [`ApplicationFailure.fromError`](https://typescript.temporal.io/api/classes/common.applicationfailure/#fromerror)
+
+- Expose additional console methods to workflow context ([#831](https://github.com/temporalio/sdk-typescript/pull/831))
+
+  `console.[error|warn|info|debug]` can now be called from Workflow code, in addition to `console.log`
+
+### Bug Fixes
+
+- [`workflow`] Make breakpoints work inside workflow isolate context ([#819](https://github.com/temporalio/sdk-typescript/pull/819))
+
+  ⚠️ NOTE: Bundles created with `bundleWorkflowCode` should only be used for calling `Worker.create` when the exact same version of `@temporalio/worker` is used. (If you don't pin to exact versions in your `package.json`, then you should use a lockfile, and both the machine that runs `bundleWorkflowCode` and `Worker.create` should run `npm ci`, not `npm install`.)
+  
+- Avoid using dynamic import in `@temporalio/testing` ([#805](https://github.com/temporalio/sdk-typescript/pull/805))
+- [`worker`] Don't start activity poller if no activities registered ([#808](https://github.com/temporalio/sdk-typescript/pull/808))
+- Update `proto3-json-serializer` to `^1.0.3` ([#809](https://github.com/temporalio/sdk-typescript/pull/809))
+- Help protobufjs find `long` in Yarn3 ([#810](https://github.com/temporalio/sdk-typescript/issues/810)) ([#814](https://github.com/temporalio/sdk-typescript/pull/814))
+- Add `@types/long` to client ([#735](https://github.com/temporalio/sdk-typescript/pull/735))
+- [`worker`] Improve worker default options heuristics ([#802](https://github.com/temporalio/sdk-typescript/pull/802))
+- [`worker`] Remove unnecessary `ReplayWorkerOptions` ([#816](https://github.com/temporalio/sdk-typescript/pull/816))
```

Comment:
Technically backwards incompatible but I don't think it's going to matter much...